storage/inchi/src/test/java/org/openscience/cdk/smiles/AbsoluteSMILESTest.java

```diff
@@ -88,6 +89,28 @@ void bispropenyloctatriene() throws Exception {
                 "C\\C=C/C(/C=C/C)=C(/C=C/C)\\C=C/C");
     }
 
+    @Test
+    public void testLargeMoleculeCanon() throws CDKException {
+        SmilesParser smipar = new SmilesParser(DefaultChemObjectBuilder.getInstance());
```

Comment:
Oh, actually, it's a really small patch.

Is the use of the `DefaultChemObjectBuilder` important here?